crates/node/src/command.rs

```diff
@@ -292,9 +292,22 @@ pub fn run() -> sc_cli::Result<()> {
                 }
             };
 
+            let l1_messages_config = match cli.run.l1_messages {
+                Some(ref config_path) => {
+                    if !config_path.exists() {
+                        return Err("L1 Messages Worker config file not available".into());
+                    }
+                    Some(config_path.clone())
+                }
+                None => {
+                    log::info!("madara initialized w/o L1 Messages Worker");
```

Comment:
I think this should be a `warning` for any madara node which is not running in `dev` mode

Reply:
It can be a `warning` anytime.